src/pages/home/components/send-button.tsx

```diff
@@ -1,26 +1,32 @@
-import { ScreenPaths } from '@common/types';
 import React, { memo } from 'react';
 import { useTransferableAssets } from '@store/assets/asset.hooks';
 import { WalletPageSelectors } from '@tests/page-objects/wallet.selectors';
-import { TxButton } from './tx-button';
+import { BuyTxButton, SendTxButton } from './tx-button';
 
 const SendButtonSuspense = () => {
   const assets = useTransferableAssets();
   const isDisabled = !assets || assets?.length === 0;
-  return (
-    <TxButton
-      isDisabled={isDisabled}
-      path={ScreenPaths.POPUP_SEND}
-      data-testid={WalletPageSelectors.BtnSendTokens}
-      kind="send"
-    />
-  );
+  return <SendTxButton isDisabled={isDisabled} data-testid={WalletPageSelectors.BtnSendTokens} />;
 };
-const SendButtonFallback = memo(() => (
-  <TxButton isDisabled path={ScreenPaths.POPUP_SEND} kind="send" />
-));
-export const SendButton = () => (
+
+const SendButtonFallback = memo(() => <SendTxButton isDisabled />);
+
+const SendButton = () => (
   <React.Suspense fallback={<SendButtonFallback />}>
     <SendButtonSuspense />
   </React.Suspense>
 );
+
+const BuyButtonFallback = memo(() => <BuyTxButton isDisabled />);
+
+const BuyButton = () => (
+  <React.Suspense fallback={<BuyButtonFallback />}>
+    <BuyTxButton />
+  </React.Suspense>
+);
+
+export const SendBuyButton = () => {
+  const assets = useTransferableAssets();
+  const totalBalance = assets.reduce((acc, asset) => asset.balance.toNumber() + acc, 0);
```

Comment:
Nice use of reduce, but don't we already calculate whether or not the user has assets 0 assets already?

Reply:
I will check